### PR TITLE
docs(log): improve docs on log serialization format

### PR DESCRIPTION
**Pull request by jschmid1**
### Description

* Order example log entry for increased readability
* Align explanation paragraph with this order
* Fix and improve description of log fields
* Added missing fields

KAG-157, KAG-158

### Checklist 

- [x] Review label added 
- [ ] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by Guaris** [comment]
Is this for 3.2?

File: app/_includes/md/plugins-hub/json-object-log.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-* `consumer` contains the authenticated Consumer (if an authentication plugin has been enabled).
-* `latencies` contains some data about the latencies involved:
-  * `proxy` is the time it took for the final service to process the request.
-  * `kong` is the internal Kong latency that it took to run all the plugins.
-  * `request` is the time elapsed between the first bytes were read from the client and after the last bytes were sent to the client. Useful for detecting slow clients.
+* `latencies` contains latency data
+  * `kong` time in ms that it took to run all the plugins.
+  * `request` time in ms time elapsed between the first bytes were read from the client and the last bytes sent to the client. This is useful for detecting slow clients.
+  * `proxy` time in ms it took for the upstream to process the request.
+* `tries` contains a list of iterations made by the load balancer for this request.
```
Comment (hanshuebner) [inline app/_includes/md/plugins-hub/json-object-log.md:11]: "contains a" could be left out.

File: app/_includes/md/plugins-hub/json-object-log.md [hunk 2]
```diff
@@ -1,22 +1,23 @@
...
+  * `balancer_start` unix timestamp for when the balancer started
```
Comment (hanshuebner) [inline app/_includes/md/plugins-hub/json-object-log.md:12]: Apparently, we want full stops after each item in a bullet list, so add them here.

**Review by hanshuebner — CHANGES_REQUESTED** [review]

**Comment by Guaris** [comment]
Changed the base of the branch to 3.2

**Review by Guaris — APPROVED** [review]

**Review by Guaris — APPROVED** [review]
LGTM
Checked it here https://deploy-preview-5050--kongdocs.netlify.app/hub/kong-inc/file-log/

**Pull request merged**